src/EnergyPlus/Furnaces.cc

```diff
@@ -12636,7 +12636,7 @@ namespace Furnaces {
             state.dataFurnaces->Furnace(FurnaceNum).MinOATCompressorCooling =
```

Comment:
Also, looking up at this function, is it necessary to pass in `FurnaceName`? Isn't that `thisFurnace.Name`?